packages/backend/src/configuration/logger.ts

```diff
@@ -1,30 +1,31 @@
 import log4js from 'log4js';
 import { config } from './environment';
+import { RequestHandler } from 'express';
 
 const configLogger = {
-    appenders: {
-      default: {
-        type: 'stdout',
-        layout: {
-          type: 'pattern',
-          pattern: '[%p] %m',
-      }},
-      file: {
-        type: 'file',
-        filename: 'log/server.log',
-        maxLogSize: 10485760,
-        layout: {
-          type: 'pattern',
-          pattern: '[%d{ISO8601}] [%p] %m',
-      }
-    }},
-    categories: {
-      all: { appenders: ['default', 'file'], level: 'all'},
-      default: { appenders: ['default'], level: 'all'},
-    }
+  appenders: {
+    default: {
+      type: 'stdout',
+      layout: {
+        type: 'pattern',
+        pattern: '[%p] %m',
+      } },
+    file: {
+      type: 'file',
+      filename: 'log/server.log',
+      maxLogSize: 10485760,
+      layout: {
+        type: 'pattern',
+        pattern: '[%d{ISO8601}] [%p] %m',
+      },
+    } },
```

Comment:
```suggestion
      },
    },
    file: {
      type: 'file',
      filename: 'log/server.log',
      maxLogSize: 10485760,
      layout: {
        type: 'pattern',
        pattern: '[%d{ISO8601}] [%p] %m',
      },
    },
  },
```
Is this weird indentation intended? :smile: